### PR TITLE
[RepairManager] improve repairmanager docker images

### PR DESCRIPTION
**Pull request by debbie-alaine**
Currently ubuntu base image is smaller than python base image. 
This Dockerfile change decreases image size from 965MB to 173MB.

@xudifsd @Anbang-Hu @YinYangOfDao

### CODE REVIEW

File: src/docker-images/RepairManagerAgent/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 MAINTAINER Deborah Sandoval <Deborah.Sandoval@microsoft.com>
 
 RUN apt-get update && apt-get --no-install-recommends install -y \
+    python3.7 \
+    python3-pip \
     systemd-sysv
 
-RUN pip3 install wheel setuptools requests pyyaml
+RUN pip3 install wheel setuptools requests
+RUN pip3 install pyyaml
```
Comment (xudifsd) [inline src/docker-images/RepairManagerAgent/Dockerfile:10]: Why not merge with above line?

Comment (debbie-alaine) [reply src/docker-images/RepairManagerAgent/Dockerfile:10]: pyyaml has a dependency on setuptools and error would occur during docker build

Comment (xudifsd) [reply src/docker-images/RepairManagerAgent/Dockerfile:10]: ok, you should add a comment

Comment (debbie-alaine) [reply src/docker-images/RepairManagerAgent/Dockerfile:10]: image

**Pull request merged**